python/simple/v1/README.md

```diff
@@ -0,0 +1,33 @@
+# Ampli Python Wrapper
+An example app using the Ampli Python Wrapper using basic Python
+
+# Usage
+
+### Setup the project
+You will need to do the following before running the app.
+
+`pip install -r requirements.txt`
+
+`cp .env.example .env`
+
+`cp ../../plugins/v1/plugins.py plugins.py`
```

Comment:
As discussed to make this easier for users lets move the plugins back into the individual examples.

I think in the future we can find a better way to share this common code across examples.